### ISSUE TITLE
RDKFuncs.chargeParent() core dumps when standardization is skipped

### ISSUE THREAD

**Issue by tdudgeon** [rdkit/rdkit#2970]
Release_2019_09 with Java wrappers, Java 11.

**Description:**

When using the `RDKFuncs.chargeParent()` method I get a core dump when the `skip_standardize` boolean parameter is set to true.

```
package org.example;


import org.RDKit.CleanupParameters;
import org.RDKit.RDKFuncs;
import org.RDKit.RWMol;

public class StandardizeExample {

    static {
        System.loadLibrary("GraphMolWrap");
    }

    private static CleanupParameters DEFAULT_CLEANUP_PARAMS = RDKFuncs.getDefaultCleanupParameters();

    public static RWMol standardize(String smiles, boolean skipStandardize) {
        RWMol mol = RWMol.MolFromSmiles(smiles);
        RDKFuncs.cleanUp(mol);
        mol = RDKFuncs.chargeParent(mol, DEFAULT_CLEANUP_PARAMS, skipStandardize);
        System.out.println(mol.MolToSmiles());
        return mol;
    }

    public static void main(String[] args) {

        System.out.println("Running with standardize");
        standardize("COC=1C=CC(NC=2N=CN=C3NC=NC23)=CC1", false);

        System.out.println("Running without standardize");
        standardize("COC=1C=CC(NC=2N=CN=C3NC=NC23)=CC1", true);
    }
}

```
In that example the first invocation succeeds but the second core dumps.
As `RDKFuncs.cleanUp(mol)` has already been called the molecule will not need standardizing again.

**Comment by greglandrum** [comment]
Thanks for reporting this. I will take a look.

**Comment by greglandrum** [comment]
Fixed in #3029
